scripts/download-binaries.sh

```diff
@@ -56,21 +52,17 @@ curl "${curl_args}O" "${kb_url}" \
   && tar xzfP "${kb_tgz}" -C "${dest_dir}" --strip-components=2 \
   && rm "${kb_tgz}"
 
-# Use a binary built from master; fedv2 requires >= 1.11 due to crd
-# validation changes and the most recent stable release (beta2) is not
-# compatible.
+# Use a stable version of kube-apiserver to ensure a version >= 1.11.
 # TODO(marun) Remove when kubebuilder includes released 1.11 binaries
-ci_version="$(curl "$curl_args" https://storage.googleapis.com/kubernetes-release-dev/ci/latest-green.txt)"
-ks_tgz="kubernetes-server-linux-amd64.tar.gz"
-ks_url="https://storage.googleapis.com/kubernetes-release-dev/ci/${ci_version}/${ks_tgz}"
-curl "${curl_args}O" "${ks_url}" \
-  && tar xzf "${ks_tgz}" kubernetes/server/bin/kube-apiserver -C "${dest_dir}" --strip-components=2 \
-  && rm "${ks_tgz}"
+stable_version="$(curl "${curl_args}" https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
```

Comment:
This should already be merged in https://github.com/kubernetes-sigs/federation-v2/pull/162/files

Reply:
@gyliu513 you'll notice that there is reference to this being merged in the commit header. The first two commits of this PR are from #162